scripts/monitor_ci.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+
+# Install E2E test
+git clone https://github.com/cgewecke/metacoin-source-verify.git
+cd metacoin-source-verify
+npm install
+
+# Publishes sources to IPFS (via Infura) and deploys contracts to Goerli
+# Account key and Infura project ID are Circle CI env variable settings.
+npm run deploy:goerli
+
+# Give monitor a chance to detect and save.
+sleep 45
```

Comment:
```suggestion
sleep 145
```

I think this will need more time when recompilation for the monitor is hooked up...